packages/rum-react/src/get-with-transaction.js

```diff
@@ -47,8 +47,16 @@ function isReactClassComponent(Component) {
 function getWithTransaction(apm) {
   return function withTransaction(name, type) {
     return function(Component) {
+      const configService = apm.serviceFactory.getService('ConfigService')
+      const loggingService = apm.serviceFactory.getService('LoggingService')
+      if (!configService.isActive()) {
+        loggingService.warn(
+          `RUM agent is inactive, route-change transaction is not instrumented`
```

Comment:
The warning here is confusing since the `withTransaction` could be used on its own and not necessarily for a `route-change`!

Reply:
We could remove the logging part here completely since we already log agent is inactive from during init phase, WDYT?

Reply:
Yeah, let's remove this completely 👍 !

Reply:
done.